scripts/tests/run_test_suite.py

```diff
@@ -107,12 +107,17 @@ class RunContext:
     default=False,
     help='Internal flag for running inside a unshared environment'
 )
+@click.option(
+    '--run-yamltests-with-chip-repl',
+    default=False,
+    is_flag=True,
+    help='Run YAML tests using chip-repl based python parser')
```

Comment:
So this is in addition to running them with chip-tool, not instead of?  Should probably be documented clearly.

Reply:
Yes these are running in addition. I updated help comment to reflect that